sql/catalyst/src/test/scala/org/apache/spark/sql/types/StructTypeSuite.scala

```diff
@@ -103,4 +105,112 @@ class StructTypeSuite extends SparkFunSuite {
     val interval = "`a` INTERVAL"
     assert(fromDDL(interval).toDDL === interval)
   }
+
+  test("find missing (nested) fields") {
```

Comment:
If you don't mind, could you split the test function?

Reply:
ok